pkg/machines/machines.less

```diff
@@ -258,6 +258,11 @@ td > .utilization-bar-pf {
     }
 }
 
+// PF4 override: rely on ct-form for spacing; remove local margin
+.ct-form > .pf-c-expandable .pf-c-expandable__content {
+    margin-top: 0;
+}
```

Comment:
I wonder if this should move to `form-layout.less` to be more generally applicable? (Are we going to use this technique elsewhere, or is this a one-off really?)